doc/releases/changes.rst

```diff
@@ -1,711 +1,9 @@
-.. _changes:
+:orphan:
 
 Release Notes
 =============
 
-Version 5.3.0 (unreleased month day, year)
---------------------------
-- Update Vega-Lite from version 5.16.3 to version 5.17.0;
-  see `Vega-Lite Release Notes <https://github.com/vega/vega-lite/releases>`_.
+We have moved the release notes to GitHub. You'll be automatically redirected there in 3 seconds, or you can click `here <https://github.com/altair-viz/altair/releases>`_.
```

Comment:
Is there an urgent need to redirect automatically?

Reply:
No urgent need. I thought redirecting in general is good to not lose users. An additional click is always an opportunity for someone to drop out. And 3 seconds felt like enough time for me to read the message but no strong preference on how we handle this. What do you think?

Reply:
Just stumbled on pandas doing the same with around 1 second of wait time: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_json.html

Reply:
Double negative alert: I thought not to auto redirect in order to not lose users..

At least we have the same intention😀!

Reply:
I don't notice a redirect on the pandas website, it stays within the pandas documentation page isn't it?

Reply:
Ah I think I see what you mean! Same intentions but I think not redirecting might indeed be a better way to achieve this. I'll remove the automated redirect so users can decide themselves if they want to go to GitHub or stay in the Altair docs.

Re the Pandas docs: Maybe they changed that... Yesterday it showed briefly a message before redirecting. Now it directly goes to `stable/reference` instead of `stable/generated`. Anyway :)